grails-app/i18n/messages.properties

```diff
@@ -1257,6 +1257,7 @@ order.orderFrom.label=Ordered From
 order.orderHasBeenPlacedWithVendor.message=Purchase order {0} has been placed
 order.orderItems.label=Line Items
 order.orderNumber.label=Order Number
+order.purchaseOrderNumber.label=PO Number
```

Comment:
Sorry I missed this earlier. Please just use order.orderNumber.label. If/when PIH wants to change the text, they can do so using a localization entry.